packages/base64/src/decode.js

```diff
@@ -60,5 +60,18 @@ const jsDecodeBase64 = (string, name = '<unknown>') => {
   return data.subarray(0, j);
 };
 
+// The XS Base64.decode function is faster, but might return ArrayBuffer (not
+// Uint8Array).  Adapt it to our needs.
```

Comment:
Does XS return ArrayBuffer, Uint8Array, or either depending? I would find it weird if the signature varied. The conditional short might not be necessary.

Reply:
It currently returns ArrayBuffer.  I don't want to be surprised if/when it used to return Uint8Array or will do so in the future.